test/extensions/filters/http/grpc_json_transcoder/grpc_json_transcoder_integration_test.cc

```diff
@@ -241,17 +240,17 @@ class GrpcJsonTranscoderIntegrationTest
 class GrpcJsonTranscoderIntegrationTestWithSizeLimit1024
     : public GrpcJsonTranscoderIntegrationTest {
 protected:
-  uint32_t maxMessageSize() const override { return 1024; }
+  uint32_t maxBodySize() const override { return 1024; }
```

Comment:
This is not good.  It basically duplicating the integration tests 4 times.  it is already very slow,  now it is 4 time slower.    Just simply testing smaller max_body_size did not impact the all tests.   This is not good.

Reply:
I see.  Thanks